From 1514432a3837d7feaf70e34fbfafc8137e539ea8 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 21 Jan 2022 18:57:21 +0100
Subject: [PATCH] hwdb: Add Microsoft Surface Pro 1 chassis quirk

See https://github.com/systemd/systemd/issues/7390#issuecomment-345546127

(cherry picked from commit bdf182debe715ff7876c610b0f93540437cb8b43)

Related: #2087778
---
 hwdb.d/20-dmi-id.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/20-dmi-id.hwdb b/hwdb.d/20-dmi-id.hwdb
index a614473bd9..c7bf6cfab1 100644
--- a/hwdb.d/20-dmi-id.hwdb
+++ b/hwdb.d/20-dmi-id.hwdb
@@ -4,3 +4,7 @@
 dmi:bvnLENOVO*
     ID_SYSFS_ATTRIBUTE_MODEL=product_version
     ID_VENDOR_FROM_DATABASE=Lenovo
+
+# Microsoft Surface 1's chassis type
+dmi:bvnMicrosoft Corporation*:pvrSurface with Windows 8 Pro*
+    ID_CHASSIS=tablet
